lib/bundler/cli/inject.rb

```diff
@@ -31,7 +31,7 @@ def run
 
       if added.any?
         Bundler.ui.confirm "Added to Gemfile:"
-        Bundler.ui.confirm added.map {|g| "  #{g}" }.join("\n")
+        Bundler.ui.confirm added.map {|g| "  #{g}, group => #{g.groups.inspect}, :source => '#{g.source.dump}'" }.join("\n")
```

Comment:
Thanks for review. I have added new msg in [this commit](https://github.com/bundler/bundler/pull/5456/commits/cfe4eccfc33a8f6372a7197ffae95c15e023df95)